Alerting: Add support for configuring avatar URL for the Discord notifier (#33355)
grafana/grafana: docs/sources/alerting/notifications.md

6. There will be a Webhook URL in the panel, looks like this: https://oapi.dingtalk.com/robot/send?access_token=xxxxxxxxx. Copy this URL to the Grafana DingTalk setting page and then click "finish".

### Discord

> To set up Discord, you need to create a Discord channel webhook. You can follow
> [Intro to Webhooks](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks) for a guide to do that.
> 

Reviewer: ```suggestion
To set up Discord, you must create a Discord channel webhook. For instructions on how to create the channel, refer to 
[Intro to Webhooks](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks) f.

```

Setting | Description
---------- | -----------
Webhook URL | Discord webhook URL.
Message Content | Mention a group using @ or a user using <@ID> when notifying in a channel.

> Avatar URL | Optionally provide a URL to an image to use as the avatar for the bot's message.

Reviewer: ```suggestion
Avatar URL | Optionally, provide a URL to an image to use as the avatar for the bot's message.
```

> Alternatively, you may use the [Slack](#slack) notifier by appending `/slack` to a Discord webhook URL.

Reviewer: ```suggestion
Alternately, use the [Slack](#slack) notifier by appending `/slack` to a Discord webhook URL.
```
